docs/docs/webchat/webchat-connectivity.md

```diff
@@ -0,0 +1,53 @@
+---
+id: webchat-connectivity
+title: Connectivity
+---
+
+Sometimes, the connection can be lost between the server and the chatbot. To handle this situation, you can set specific parameters to quickly detect the connection issue.
```

Comment:
To handle this situation, you can set specific parameters to tune how to detect the connection issue.